### PR TITLE
ADEN-2463 Stabilise marker test

### PR DESCRIPTION
**Pull request by nandy-andy**
The `TestExtraMarker_GHOST` was failing every day for 3 days in a row. In the changes below we fixed it with `WebDriverWait` and renamed classes and methods to fit our team/internal convention.

I'm creating this pull request because accidentally I've closed the previous one opened by @drets:
https://github.com/Wikia/selenium-tests/pull/877

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsExtraMarkerOasis.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      groups = "AdsExtraMarkerOasis",
+      dataProvider = "adsExtraMarkerOasis"
+  )
+  public void AdsExtraMarkerOasis(final String wikiName,
+                                  final String article,
+                                  final String adType,
+                                  final String message) {
+    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    final AdsBaseObject wikiPage = new AdsBaseObject(driver, testedPage);
+    new WebDriverWait(driver, 30).until(new ExpectedCondition<Boolean>() {
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsExtraMarkerOasis.java]: Magic number! Is it 30 seconds? Can we put it in a const with a descriptive name?

**Comment by nandy-andy** [comment]
LGTM since we can't wait for a specific element on a page we have to wait some time (`30`). I just left a minor comment please address it and merge the changes :+1:

**Comment by drets** [comment]
> since we can't wait for a specific element on a page we have to wait some time (30).

@nandy-andy: we don't wait _some time (30)_. 
What we do is: we wait (up to 30 sec) until we will have specific text in browser logs, if not - test fails.

**Comment by nandy-andy** [comment]
That's what I've meant! Thanks for clarification and removing magic number from the code :japanese_ogre: 

LGTM :+1:

**Comment by WikiaTech** [comment]
SonarQube analysis reported 30 issues:
* MINOR 30 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "FFFFFF" 8 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L200) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "wikia" 7 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L80) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "naruto" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L46) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "MH3U:_Monsters" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L102) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "ru.elderscrolls" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L98) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Interstitial/NotScalable" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L523) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/ProvidersChain" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L405) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "DirectGpt; RemnantGpt; Liftium" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L406) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "CORP_TOP_LEADERBOARD" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L80) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Interstitial" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L516) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "HUB_TOP_LEADERBOARD" 8 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L84) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "%D0%9A%D0%B2%D0%B5%D1%81%D1%82%D1%8B_%28Skyrim%29" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L98) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "wka.hub/_ent_hub//hub" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L88) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "AAAAAA" 9 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L199) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "src/test/resources/adsResources/wikia_skin_right.png" 9 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L198) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "126608052" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L165) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest" 44 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L34) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "zh.pad" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L54) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Wikia" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L80) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "leagueoflegends" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L47) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "TOP_LEADERBOARD" 10 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L165) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "TOP_RIGHT_BOXAD" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L177) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Krux/Page_1" 4 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L443) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest-pad" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L505) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "elderscrolls" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L68) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest-pluto" 8 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L104) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "gameshub" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L84) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "monsterhunter" 5 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L102) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "src/test/resources/adsResources/wikia_skin_left.png" 9 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L197) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "yugioh" 3 times.](https://github.com/Wikia/selenium-tests/blob/b34740d43775accc053a24309308b7f63449f69e/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L45) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)

**Pull request merged**